src/deroff.rs

```diff
@@ -43,7 +44,7 @@ struct Deroffer {
     tblstate: TblState,
     tblTab: String,
     eqn: bool,
-    output: String,
+    output: Cell<String>,
```

Comment:
> @pickfire 4959b54 wasn't meant to be on this branch, and 0af9b19 is on here already

https://github.com/rust-dc/fish-manpage-completions/commit/4959b5407c4d3153701bbdc81efe6519679db698 is here, I thought wasn't meant to be on this branch?

Reply:
Okay, I guess this isn't that important. I will just ignore this.